Description: Upstream changes introduced in version 2010.09~rc2-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 u-boot (2010.09~rc2-1) unstable; urgency=low
 .
   * New upstrem release candidate.
 .
 The person named in the Author field signed this changelog entry.
Author: Clint Adams <schizo@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- u-boot-2010.09~rc2.orig/snapshot.commit
+++ u-boot-2010.09~rc2/snapshot.commit
@@ -1 +1 @@
-07517e7f4f582d1607adbc444bab75af3ac4a7f9  Sun, 19 Sep 2010 17:47:52 +0200
+$Format:%H  %cD$
